docs/docs/APIdocs/layers/nn/Merge Layers/Abs.md

```diff
@@ -0,0 +1,37 @@
+## Abs ##
```

Comment:
you can just leave it there and I will confirm with Yiheng and helen about layer folders and adjust later.